devicemodel/hw/vga.c

```diff
@@ -0,0 +1,1428 @@
+/*
+ * Copyright (C) 2022 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-FreeBSD
```

Comment:
This license identifier is deprecated (see https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html). Please use `BSD-2-Clause-Views` instead. See https://spdx.org/licenses/BSD-2-Clause-Views.html for more details.